TombEngine/Objects/TR3/Entity/tr3_shiva.cpp

```diff
@@ -21,16 +22,18 @@ using std::vector;
 
 namespace TEN::Entities::Creatures::TR3
 {
-	constexpr auto SHIVA_GRAB_ATTACK_DAMAGE		= 150;
+	constexpr auto SHIVA_WALK_TURN_RATE_MAX = ANGLE(4.0f);
+	constexpr auto SHIVA_ATTACK_TURN_RATE_MAX = ANGLE(4.0f);
+	constexpr auto SHIVA_DOWNWARD_ATTACK_RANGE = SQUARE(SECTOR(1));
+	constexpr auto SHIVA_GRAB_ATTACK_RANGE = SQUARE(SECTOR(1.25f));
+	constexpr auto SHIVA_GRAB_ATTACK_DAMAGE = 150;
 	constexpr auto SHIVA_DOWNWARD_ATTACK_DAMAGE = 180;
-
-	constexpr auto LARA_ANIM_SHIVA_DEATH = 7;
-
-	#define SHIVA_WALK_TURN_RATE_MAX   ANGLE(4.0f)
-	#define SHIVA_ATTACK_TURN_RATE_MAX ANGLE(4.0f)
+	constexpr auto SHIVA_MESH_SWITCH_TIME = 50000;
```

Comment:
What is `SHIVA_MESH_SWITCH_TIME`? It's unused.

Reply:
Will remove it.

Reply:
Fixed